CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ since 4.7 and prior.
   - `--admission-controller-timeout`
   Using them has no effect.
 - The current hierarchical implementation for defining Collections is deprecated and will be replaced by a more comprehensive search-based definition in the future.
+- The manifest install method is now deprecated and will be removed in the future.
```

Comment:
This is a bit terse. Some users might not be aware that this is what they are using.
Perhaps we should:
- mention the roxctl commands and UI pages that are involved
- mention that users should switch to operator (if available) or helm